about/contributors.md

```diff
@@ -0,0 +1,25 @@
+## CloudEvents contributors
+
+* **Amazon**
+  * [AWS Lambda events](https://docs.aws.amazon.com/lambda/latest/dg/invoking-lambda-function.html)
+  * Arun Gupta, Ajay Nair, Rob Leidle, Orr Weinstein
+* **Google**
+  * [Google Cloud Functions](https://cloud.google.com/functions/) and [Cloud Functions for Firebase](https://firebase.google.com/docs/functions/)
+  * [Sarah Allen](https://github.com/ultrasaurus), [Rachel Myers](https://github.com/rachelmyers) (Serverless WG)
```

Comment:
I don't see the pattern here w.r.t. when a new bullet is used. Can we either put all people from one company on the same bullet or give each person their own bullet?

Reply:
Products are listed at the same level that people are. Not all have a product. Should they be listed? Are products contributors?

Reply:
I'd prefer if we didn't list them.  And to be honest, I'm not sure I want to even have a "people list". If people want to see who contributed then they can look at the git logs.  No different than a git code repo.

Reply:
In my opinion, our big challenge is to gain traction with this effort, otherwise it will fade away and our wonderful effort will be for nothing.  Fortunately, we have some huge assets to help us on our journey: *the CNCF* and *the quality of people who are working on this already*.  

The fact that so many incredible people (and orgs) are collaborating on this sends a clear signal to the market that something of significance is happening.  Making these names hard to access is akin to removing the marquee customers on the landing page of your product or hiding the celebrity names on a movie poster.  I believe we will be handicapping ourselves in a major way by not making these people clearly visible.  Further, maintaining a Contributor list that is clearly visible shows that we welcome, recognize and celebrate our contributors, and it will be helpful in seeking new contributors.  We should therefore make these names as easy to see as possible and a Contributor list is a great way to do this.

Being explicit about these methods always feels awkward.  But building products/traction/community is a science.  It will take great product, but also great packaging.

Reply:
That said, we need to make sure these people are ok with being listed.  Including a clear statement on this page that says: **“Contributions do not constitute an official endorsement”** will greatly help here.

Also, listing products feels like it will add a higher burden of maintenance and complexity.  Already, there are some products and team members listed here that are out of date.  Maybe you can optionally allow for people to put their name and the product they work on?  Also, perhaps their twitter handle so that others can contact them?

**Example:**

* **Serverless Inc.**
   * Austen Collins - Serverless Framework - @austencollins

Reply:
What's the bar for being added? Company, product or people-wise ?

Reply:
@duglin Perhaps it's as simple as they contribute to the Github repository.

Reply:
maybe. Just feels like we're setting the board up for a game  :-) 

I'm not keen on listing products - I don't see the point of that. Products are not contributing, companies and people are.

Reply:
The products provide context for why the companies are contributing, and actually more relevant than the companies themselves.

Reply:
If the bar is contributing, I should be listed. Not arguing either way, but it's an omission if that is the bar.

Reply:
@notque picking this back up... I will add you in my updated PR!